docs/basic-features/data-fetching/overview.md

```diff
@@ -23,6 +23,7 @@ description: 'Next.js allows you to fetch data in multiple ways, with pre-render
     <li><a href="https://github.com/vercel/next.js/tree/canary/examples/cms-graphcms">GraphCMS Example</a> (<a href="https://next-blog-graphcms.vercel.app/">Demo</a>)</li>
     <li><a href="https://github.com/vercel/next.js/tree/canary/examples/cms-kontent">Kontent Example</a> (<a href="https://next-blog-kontent.vercel.app/">Demo</a>)</li>
     <li><a href="https://static-tweet.vercel.app/">Static Tweet Demo</a></li>
+    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/with-enterspeed">Enterspeed Example</a> (<a href="https://next-blog-demo.enterspeed.com/">Demo</a>)</li>
```

Comment:
We should probably reference the same app as in the README to avoid things get out of sync. See https://github.com/vercel/next.js/pull/35897/files#diff-38f26457281442e4cb469903b892b919f46d7867462f9278e15c105774e27ebcR7